src/components/Articles.jsx

```diff
@@ -0,0 +1,41 @@
+import { useEffect, useState } from "react";
+// import { useParams } from "react-router-dom";
+import { getArticles} from "./api";
+
+const Articles = () => {
+    const [articleList, setArticleList] = useState([])
+    // const { article_name } = useParams();
+    
+    useEffect(() => {
+        console.log('hello')
```

Comment:
Remember to get rid of console logs before PRs, check through the logs and make sure you get rid of them as you go along 🙂